asapdiscovery-data/asapdiscovery/data/postera/postera_uploader.py

```diff
@@ -77,6 +97,27 @@ def push(self, df) -> None:
                     smiles_field=self.smiles_field,
                     overwrite=self.overwrite,
                 )
+        return df_copy, molset_name, new_molset
+
+    @staticmethod
+    def join_with_manifold_data(original, molset_query_df):
+        """
+        Join the original dataframe with manifold data
+        that is returned from a query to the manifold API
+        """
+        data = original.copy()
+        subset = molset_query_df[["id", "smiles"]]
```

Comment:
(I'll be honestly I hardly know what's going on at this point but...) does it make sense to have these hard-coded? always a concern

Reply:
This is defined in the Manifold API docs [here](https://api.postera.ai/api/v1/docs/#tag/moleculesets/operation/moleculesets_get_all_molecules_retrieve), so I think it is pretty stable but can wrap a little bit

Reply:
Okay I wrapped it a bit to avoid issues, good pickup.